prow/config.yaml

```diff
@@ -102,7 +102,7 @@ tide:
     - kubeflow/pytorch-operator
     - kubeflow/testing
     - kubeflow/tf-operator
-    - kubeflow/hp-tuning
+    - kubeflow/katib
```

Comment:
Can you also add all the repositories that are currently missing

And 
kubeflow/experimental-kvc
kubeflow/experimental-seldon
kubeflow/experimental-beagle
kubeflow/caffe2-operator
kubeflow/reporting
kubeflow/website

Related to kubeflow/website#4

Reply:
OK, I will add it.